.github/labeler.yml

```
@@ -0,0 +1,16 @@
+🚀 feat:
+  - '^(\[feat\])'
+🐞 fix:
+  - '^(\[fix\])'
+🔨 refactor:
+  - '^(\[refactor\])'
+🎨 style:
+  - '^(\[style\])'
+✅ test:
+  - '^(\[test\])'
+⚙️ utils:
+  - '^(\[utils\])'
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)